kod/object/passive/quest.kod

```diff
@@ -605,6 +605,11 @@ SetQuestNode( node_object = $ )
 	return;
 }
 
+GetQuestNode()
```

Comment:
I think the main question is this one---whether the quest should expose its quest node to the outside world.  My intuition is not to do this, as it makes even more complexity possible, as now anything outside the quest engine can start doing arbitrary things with quests, without the engine knowing.

The alternative would be to take whatever you're currently using this for (DeadlineExpired?) and move it into this file instead, as "ExpireQuest" or something like that.

Reply:
Makes sense and confirms a suspicion of mine, thanks 👍🏻. I'll refactor.

Reply:
I've addressed this in the latest commit.